### PR TITLE
Android app to list and see details from upcoming movies

### PR DESCRIPTION
**Pull request by leofin**
First of all, I had a problem with the commits and github. I started git from the console and did not do commits or pushs in the master branch, when I went to see github, there was no branch master and prevented from creating a PR :(

So I ended up deleting the previous repository and starting one from scratch, I deleted the git folder from the project and started again, but when I finished it all.

When I came to review my own PR, I came across all the files of the project, those 900 and few that you should be enjoying there.

In short, this PR is not revisable by file size and types.

I put a SS of the files path in the readme for easy review.

Sorry for the inconvenience.

### CODE REVIEW

File: app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Genre.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.fin.leo.movieholics.TMDb.entity;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Genre.java]: Entity Genre.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Genres.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.fin.leo.movieholics.TMDb.entity;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Genres.java]: Entity Genres.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Movie.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package com.fin.leo.movieholics.TMDb.entity;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Movie.java]: Entity Movie.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Upcoming.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package com.fin.leo.movieholics.TMDb.entity;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/TMDb/entity/Upcoming.java]: Entity Upcoming.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/TMDb/services/API.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
+package com.fin.leo.movieholics.TMDb.services;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/TMDb/services/API.java]: API.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/app/models/Movie.java [hunk 6]
```diff
@@ -0,0 +1,52 @@
+package com.fin.leo.movieholics.app.models;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/app/models/Movie.java]: Movie.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/app/services/UpcomingServices.java [hunk 7]
```diff
@@ -0,0 +1,77 @@
+package com.fin.leo.movieholics.app.services;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/app/services/UpcomingServices.java]: UpcomingServices.java

Just to show my class. :)

File: app/src/main/java/com/fin/leo/movieholics/app/views/MainActivity.java [hunk 8]
```diff
@@ -0,0 +1,36 @@
+package com.fin.leo.movieholics.app.views;
```
Comment (leofin) [inline app/src/main/java/com/fin/leo/movieholics/app/views/MainActivity.java]: MainActivity.java

Just to show my class. :)
